data/siteText.yaml

```diff
@@ -97,7 +97,11 @@ landUse-p2: >-
   Fragments, films, and foams were more frequent in samples from more urban basins
 
 ## Footer ##
-footer-text: Footer Text
+footer-text1: U.S. Department of the Interior |
```

Comment:
I wouldn't put '|' in the text here, rather it is a separator that can go directly into the html